### ISSUE TITLE
Large day numbers of the Calendar plugin

### ISSUE THREAD

**Issue by pyxelr** [kepano/obsidian-minimal#579]
### Is the bug present when using the default Obsidian theme?

No

### Is the bug present when snippets and plugins are disabled?

Yes

### Minimal theme version

6.3.5

### Describe the bug

After the recent upgrade to `6.3.4` or `6.3.5` minimal theme version, the day numbers of the popular [Calendar plugin](https://github.com/liamcain/obsidian-calendar-plugin) became much bigger:

![CleanShot 2023-06-18 at 23 32 44](https://github.com/kepano/obsidian-minimal/assets/30161753/7f4ab654-f9c6-4442-8435-d7cd00c3dd5d)

Would it be possible to revert this size change, as it is for example in the Default Obsidian theme?

### Debug info

```text
SYSTEM INFO:
	Obsidian version: v1.3.5
	Installer version: v1.3.5
	Operating system: Darwin Kernel Version 22.5.0: Mon Apr 24 20:53:19 PDT 2023; root:xnu-8796.121.2~5/RELEASE_ARM64_T6020 22.5.0
	Login status: logged in
	Catalyst license: vip
	Insider build toggle: off
	Live preview: on
	Legacy editor: off
	Base theme: dark
	Community theme: Minimal v6.3.5
	Snippets enabled: 0
	Restricted mode: off
	Plugins installed: 18
	Plugins enabled: 18
		1: Text Snippets v0.1.2
		2: Templater v1.16.0
		3: Style Settings v1.0.6
		4: Minimal Theme Settings v6.3.1
		5: Latex Suite v1.7.2
		6: Icon Folder v2.1.2
		7: Execute Code v1.8.1
		8: Emoji Shortcodes v2.2.0
		9: Copy Block Link v1.0.4
		10: Filename Heading Sync v1.9.0
		11: Natural Language Dates v0.6.1
		12: Find orphaned files and broken links v1.9.0
		13: Recent Files v1.3.6
		14: Linter v1.15.2
		15: Smart Typography v1.0.18
		16: Paste URL into selection v1.7.0
		17: Advanced Tables v0.18.1
		18: Calendar v1.5.10

RECOMMENDATIONS:
	Custom theme and snippets: for cosmetic issues, please first try updating your theme and disabling your snippets. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community theme and snippets.
	Community plugins: for bugs, please first try updating all your plugins to latest. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community plugins.
```

**Comment by kepano** [comment]
closed in 6.3.6

**Comment by pyxelr** [comment]
Unfortunately, it looks like the issue continues to occur in `6.3.8`, and is also reported here: https://github.com/kepano/obsidian-minimal/issues/580

**Comment by kepano** [comment]
Fixed in 6.3.9
